crates/viewer/re_space_view_spatial/src/view_3d.rs

```diff
@@ -443,3 +446,84 @@ impl SpaceViewClass for SpatialSpaceView3D {
         self.view_3d(ctx, ui, state, query, system_output)
     }
 }
+
+fn view_property_ui_grid3d(
+    ctx: &ViewerContext<'_>,
+    ui: &mut egui::Ui,
+    view_id: SpaceViewId,
+    fallback_provider: &dyn re_viewer_context::ComponentFallbackProvider,
+    view_state: &dyn SpaceViewState,
+) {
+    let property = ViewProperty::from_archetype::<LineGrid3D>(
+        ctx.blueprint_db(),
+        ctx.blueprint_query,
+        view_id,
+    );
+    let Some(reflection) = ctx.reflection.archetypes.get(&property.archetype_name) else {
+        ui.error_label(format!(
+            "Missing reflection data for archetype {:?}.",
+            property.archetype_name
+        ));
+        return;
+    };
+
+    let query_ctx = property.query_context(ctx, view_state);
+    let sub_prop_ui = |ui: &mut egui::Ui| {
+        for field in &reflection.fields {
+            // TODO(#1611): The color picker for the color component doesn't show alpha values so far since alpha is almost never supported.
+            // Here however, we need that alpha color picker!
```

Comment:
Note: another short term quick win by rolling tagged component to view property. We could make the component ui editor white list some Archetype/Field for the transparency ui.